### ISSUE TITLE
Transient Analysis is declared as Static

### ISSUE THREAD

**Issue by gtzortzo** [ansys/pydpf-core#959]
### Before submitting the issue

- [X] I have checked for [Compatibility](https://dpf.docs.pyansys.com/version/stable/getting_started/compatibility.html#id1) issues
- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

When I load a .rth file from a Transient Thermal analysis in PyDPF, I noticed that it assumes it is a static analysis. 
Is this a known issue?

I double-checked with Transient Structural analysis too.

Here you can find a dummy_thermal.rth and a dummy_structural.rst files for reproducing the issue:
[dummy_files.zip](https://github.com/ansys/pydpf-core/files/11518967/dummy_files.zip)

### Steps To Reproduce

- Put dummy_thermal.rth and dummy_structural.rst into the same directory with the running Python script
- Execute the following lines of code:

```
from ansys.dpf import core as dpf

filename = "dummy_thermal.rth" # or "dummy_structural.rst"
model = dpf.Model(filename)
print(model)
```

- Check output:

```
DPF Model
------------------------------
Static analysis
```

### Which Operating System causes the issue?

Windows

### Which DPF/Ansys version are you using?

DPF Server 2023.2.pre1

### Which Python version causes the issue?

3.11

### Installed packages

Package                  Version
------------------------ --------
ansys-dpf-core           0.8.1
ansys-dpf-gate           0.3.1
ansys-dpf-gatebin        0.3.1
ansys-grpc-dpf           0.7.1
asttokens                2.2.1
backcall                 0.2.0
cachetools               5.3.0
certifi                  2023.5.7
charset-normalizer       3.1.0
colorama                 0.4.6
comm                     0.1.3
contourpy                1.0.7
cycler                   0.11.0
debugpy                  1.6.7
decorator                5.1.1
executing                1.2.0
fonttools                4.39.4
google-api-core          2.11.0
google-api-python-client 2.86.0
google-auth              2.18.1
google-auth-httplib2     0.1.0
googleapis-common-protos 1.59.0
grpcio                   1.54.2
httplib2                 0.22.0
idna                     3.4
importlib-metadata       6.6.0
ipykernel                6.23.1
ipython                  8.13.2
jedi                     0.18.2
jupyter_client           8.2.0
jupyter_core             5.3.0
kiwisolver               1.4.4
matplotlib               3.7.1
matplotlib-inline        0.1.6
nest-asyncio             1.5.6
numpy                    1.24.3
packaging                23.1
parso                    0.8.3
pickleshare              0.7.5
Pillow                   9.5.0
pip                      23.1.2
platformdirs             3.5.1
pooch                    1.7.0
prompt-toolkit           3.0.38
protobuf                 4.23.1
psutil                   5.9.5
pure-eval                0.2.2
pyasn1                   0.5.0
pyasn1-modules           0.3.0
Pygments                 2.15.1
pyparsing                3.0.9
python-dateutil          2.8.2
pyvista                  0.39.1
pywin32                  306
pyzmq                    25.0.2
requests                 2.30.0
rsa                      4.9
scooby                   0.7.2
setuptools               65.5.0
six                      1.16.0
stack-data               0.6.2
tornado                  6.3.2
tqdm                     4.65.0
traitlets                5.9.0
uritemplate              4.1.1
urllib3                  1.26.15
vtk                      9.2.6
wcwidth                  0.2.6
zipp                     3.15.0

**Comment by laurent-ge** [comment]
Indeed, the provided dummy_files are printed as Static, whereas they contain multiple Substeps.
This issue is actually seen in [the transient example](https://dpf.docs.pyansys.com/version/stable/examples/01-transient_analyses/00-basic_transient.html#sphx-glr-examples-01-transient-analyses-00-basic-transient-py) we have in the doc.

Note: I have a transient thermal file locally that is correctly printed as Transient. Maybe this is because I have 2 LoadSteps (with many Substeps) in this model, where the other cases above have 1 LoadStep.

@rafacanton

**Comment by PProfizi** [comment]
@oparreno I can confirm this is still the case with the latest dev version of DPF.

**Comment by PProfizi** [comment]
@ricardopeixotocoelho FYI.

**Comment by oparreno** [comment]
@laurent-ge, @PProfizi, @rafacanton 
This is expected in any version before 2024R1, regardless of the DPF server version. Please check the MAPDL version.

**Comment by PProfizi** [comment]
-> In any **MAPDL** solver version used to generate the result file, right?

Thanks @oparreno, closing this issue
